src/app/components/CreateTask.tsx

```diff
@@ -0,0 +1,287 @@
+'use client'
+
+import { PrimaryBtn } from '@/components/buttons/PrimaryBtn'
+import { SecondaryBtn } from '@/components/buttons/SecondaryBtn'
+import { StyledAutocomplete } from '@/components/inputs/Autocomplete'
+import { StyledTextField } from '@/components/inputs/TextField'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { useFocusableInput } from '@/hooks/useFocusableInput'
+import { AttachmentIcon, DoneIcon, HumansIcon, InprogressIcon, InreviewIcon, TodoIcon } from '@/icons'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+import { Close } from '@mui/icons-material'
+import { Avatar, Box, Stack, Typography } from '@mui/material'
+import { ReactNode, useState } from 'react'
+
+interface IAssignee {
+  name: string
+  type: string
+  img?: string
+}
+
+//This is mock data which will be replaced after API integration.
+const status = ['Todo', 'In Progress', 'In review', 'Done']
+
+//This is mock data which will be replaced after API integration.
+const statusIcons: { [key: string]: ReactNode } = {
+  Todo: <TodoIcon />,
+  'In Progress': <InprogressIcon />,
+  'In review': <InreviewIcon />,
+  Done: <DoneIcon />,
+}
```

Comment:
@rrojan let's make the keys type safe as well

Reply:
I have added `StatusKey` type for now but this is only to represent the mock data. This will change when API code is available and most probably we will use type defined by @rrojan.

Reply:
Please refer to `StatusTypeSchema` in `src/types/dto/status.dto.ts` for the status keys enum @aatbip 

cc @saugatmaharjan

Reply:
I will use the same type while working on the API integration @rrojan